### ISSUE TITLE
F-Droid release

### ISSUE THREAD

**Issue by rhpvorderman** [geteduroam/android-app#65]
The eduroamCAT tool is available on [F-Droid](https://f-droid.org/en/packages/uk.ac.swansea.eduroamcat/). This tool is not yet. Are there plans to do so?

**Comment by pauldekkers** [comment]
Definitely. It was on my agenda for 2024 to do after the release of the new Apps, I just didn't get to it yet ;-)

**Comment by rhpvorderman** [comment]
Awesome! Thanks! I personally run a custom ROM to avoid google and to maximize the amount of free software in my life, so this is great news. Nice that this is on the planning, even though it is a thing only a few users will care about. Thanks again!

**Comment by tibequadorian** [comment]
Any news on this?

**Comment by tibequadorian** [comment]
I opened a new issue here: https://gitlab.com/fdroid/rfp/-/issues/2890
I’ve also done some initial work on packaging here: https://gitlab.com/tibequadorian/fdroiddata/-/tree/app.eduroam.geteduroam

For inclusion, this repository should include Fastlane files: https://gitlab.com/-/snippets/1895688

While packaging, I found the versioning of this app confusing. There are many tagged snapshots without bumping the version number and going by the date for the latest update on the play store, the current stable release seems to be [2.0.1-androidsnapshot-1926](https://github.com/geteduroam/mobile-app/releases/tag/2.0.1-androidsnapshot-1926)? Proper versions would also streamline the inclusion on F-Droid.

**Comment by phikal** [comment]
> A Github release would be great too. Thus people using [Obtainium](https://github.com/ImranR98/Obtainium) could directly install from there.

I'd like to second this suggestion.

**Comment by rhpvorderman** [comment]
Great! Thanks for releasing on F-Droid. On computers it is relatively easy, but on phones it is quite hard to escape the walled gardens of Google and Apple. I previously build the app manually in Android studio, but this is much more comfortable.
